src/client/Microsoft.Identity.Client/Region/RegionDiscoveryProvider.cs

```diff
@@ -86,7 +86,7 @@ private async Task<string> GetRegionAsync(RequestContext requestContext)
 
             if (!string.IsNullOrEmpty(region))
             {
-                requestContext.Logger.Info($"[Region discovery] Region found in environment variable: {region}.");
+                requestContext.Logger.Info($"[Region Discovery] Region found in environment variable: {region}.");
```

Comment:
why a capitalized D? (this is a sentence?). Microsoft is advising to use sentence casing in public facing docs, logs etc ...

Reply:
Capitalized for consistency. We used `[Region Discovery]` and `[Instance Discovery]` else where. I wasn't aware of that guidance, so I can update to use sentence casing.

Reply:
@jmprieur - these are MSAL logs, not docs.